CONTRIBUTING.md

```
@@ -96,11 +96,21 @@ This integration follows Home Assistant's [integration quality standards](https:
-
-This project comes with a complete development environment in a container, easy to launch if you use Visual Studio Code. With this container you will have a standalone Home Assistant instance running and already configured.
-
-You can also run tests using `pytest` to ensure your changes don't break existing functionality.
+## Pull Request Process
+
+When submitting a pull request:
+
+1. Create a descriptive feature/bugfix branch from `main`.
+2. Follow the checklist provided in the [Pull Request Template](.github/pull_request_template.md).
+3. Ensure all automated validation steps pass:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)